.github/workflows/deploy.yml

```diff
@@ -0,0 +1,100 @@
+name: Build and Deploy
+concurrency: build_and_deploy_${{ github.ref_name }}
+
+on: push
+
+jobs:
+  lint:
+    name: Lint
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        tests: [rubocop, erblint, prettier]
+        include:
+          - tests: rubocop
+            command: bundle exec rubocop
+          - tests: erblint
+            command: bundle exec erblint app
+          - tests: prettier
+            command: yarn prettier --check app
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+      - name: Setup Ruby
+        uses: ruby/setup-ruby@v1
+        with:
+          ruby-version: 3.2.2
+          bundler-cache: true
+      - name: Use Node.js 20.9.0
+        uses: actions/setup-node@v3
+        with:
+          node-version: 20.9.0
+          cache: 'npm'
+      - name: Install dependencies
+        run: |
+          sudo apt-get update && sudo apt-get -yqq install libpq-dev
+          bundle install --jobs 4 --retry 3
+          bundle exec rails assets:precompile
+        env:
+            DATABASE_URL: postgres://postgres:password@localhost:5432
+            GOVUK_NOTIFY_API_KEY: fake_key
+            SIGN_IN_METHOD: persona
+      - name: ${{ matrix.tests }}
+        run: ${{ env.COMMAND }}
+        env:
+          COMMAND: ${{ matrix.command }}
+          DATABASE_URL: postgres://postgres:password@localhost:5432
+          GOVUK_NOTIFY_API_KEY: fake_key
+          SIGN_IN_METHOD: persona
+
+  test:
+    name: Test
+    runs-on: ubuntu-latest
+    services:
+        postgres:
+          image: postgres:13.5
```

Comment:
Is there a reason we are using a different version of postgres here? In the dockerfile, we have it is at 16.

I'm sure it won't make a difference to the tests, but I think it would be good to make it consistent if possible.